src/main/java/org/example/gabartischolarsqbbe/controller/JobController.java

```diff
@@ -0,0 +1,105 @@
+package org.example.gabartischolarsqbbe.controller;
+
+import jakarta.validation.Valid;
+import org.example.gabartischolarsqbbe.dto.ApiResponse;
+import org.example.gabartischolarsqbbe.dto.JobRequest;
+import org.example.gabartischolarsqbbe.dto.JobResponse;
+import org.example.gabartischolarsqbbe.service.JobService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api/jobs")
+@CrossOrigin(origins = "*")
+public class JobController {
+    
+    private final JobService jobService;
+    
+    @Autowired
+    public JobController(JobService jobService) {
+        this.jobService = jobService;
+    }
+    
+    /**
+     * Create a new job
+     * POST /api/jobs
+     * Request body: {"name": "Software Developer", "code": "SD001"}
+     */
+    @PostMapping
+    public ResponseEntity<ApiResponse<JobResponse>> createJob(@Valid @RequestBody JobRequest jobRequest) {
```

Comment:
it would be better to create a `CreateJobRequestDto` rather than accepting the param as a `JobRequest`